README.md

```diff
@@ -987,6 +987,21 @@ params do
 end
 ```
 
+In the example above Grape will use `blank?` to check if the `shelf_id` param is present.
+
+But you can use a `Proc` with custom code as well. If it returns true, the block will be evaluated.
```

Comment:
This is documentation, so keep things succinct. Just continue the previous phrase with something like "given also takes a proc". You don't need to say "if it returns true", because that's not ... true, anything truthy will work. Anyway, just try to say the most with the least amount of words :)